docs/api/datasets/non_geo_datasets.csv

```diff
@@ -49,6 +49,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `SKIPP'D`_,R,"Fish-eye","CC-BY-4.0","363,375",-,64x64,-,RGB
 `SkyScript`_,IC,"NAIP, orthophotos, Planet SkySat, Sentinel-2, Landsat 8--9",MIT,5.2M,-,100--1000,0.1--30,RGB
 `So2Sat`_,C,Sentinel-1/2,"CC-BY-4.0","400,673",17,32x32,10,"SAR, MSI"
+`SODAA`_,OD,Aerial,"CC-BY-SA","2513",9,"varying","RGB"
```

Comment:
```suggestion
`SODA`_,OD,Aerial,"CC-BY-SA","2513",9,"varying","RGB"
```
I see CC-BY-SA [here](https://gcheng-nwpu.github.io/), but I wish we knew _which_ CC-BY-SA version. I usually use a size range instead of "varying".